README.md

```diff
@@ -60,16 +77,30 @@ npm run build
 ```
 
 Install the socket server node packages. Open a new terminal, `cd` to `cla_frontend` and run:
+
+on Linux
 ```
 cd cla_frontend/cla_socketserver/
+```
+on Windows
+```
+cd cla_socketserver
```

Comment:
That doesn't look like a windows difference to me, that looks like maybe the `cd cla_frontend` bit was duplicated in the first place